### DISCUSSION TITLE
Enhancement: Discord RPC

### DISCUSSION THREAD

**Discussion by hotsno** [Syncplay/syncplay#491] [Ideas]
A cool addition would be a Discord RPC that might what you're watching, how many users are in the party, etc. Just wanted to throw the idea out there. Here's an example of what another Discord RPC looks like.

![image](https://user-images.githubusercontent.com/71658949/138153463-dbd05c43-9e9e-450f-b8f8-c54ae678a265.png)

**Comment by Et0h** [comment]
I agree it would be cool. If someone wanted to volunteer to implement such a feature as something people could enable then I expect it would be well received
